--- foiltex.ins.orig	2005-12-28 23:29:35.000000000 +0100
+++ foiltex.ins	2005-12-29 00:24:51.000000000 +0100
@@ -115,24 +115,3 @@
 
 % the sample foils file
 \generateFile{sampfoil.tex}{t}{\from{foiltex.dtx}{sample}}
-
-
-\newcommand{\iispc}{\space\space}
-\newcommand{\vspc}{\iispc\iispc\space}
-\Msg{**********************************************************************}
-\Msg{*}
-\Msg{* To finish the installation, do the following:}
-\Msg{*}
-\Msg{*\iispc 1) Copy or move the foils.cls, foil*.clo, fltfonts.def and} 
-\Msg{*\vspc     and foils.sty files to the some directory in your TEXINPUTS}
-\Msg{*\vspc     LATEXINPUTS path.  See your TeX systems documentation for}
-\Msg{*\vspc     additional steps that may be required for your local environment.}
-\Msg{*\iispc 2) Run LaTeX on the foiltex.dtx file to produce the documentation}
-\Msg{*\vspc     in dvi format. Put the resulting foiltex.dvi file and the}
-\Msg{*\vspc     foiltex.dtx somewhere in your documentation subdirectory.}
-\Msg{*\iispc 3) Put the sample file (sampfoil.tex) in the examples directory.}
-\Msg{*}
-\Msg{**********************************************************************}
-\endinput
-
-
